searchobject/src/main/java/org/jalau/at18/searchobject/model/objectrecognizer/recognizer/VerifyModelRecognizer.java

```diff
@@ -12,7 +12,6 @@
 import java.util.logging.Logger;
 
 /**
- *
```

Comment:
please add what it's the function of the method and add the paramether call in the method

Reply:
I have added more documentation to the class